core/src/broadcast_stage/broadcast_utils.rs

```diff
@@ -30,16 +30,16 @@ pub struct UnfinishedSlotInfo {
     pub parent: Slot,
 }
 
-pub(super) fn recv_slot_entries(receiver: &Receiver<WorkingBankEntry>) -> Result<ReceiveResults> {
+pub(super) fn recv_slot_entries(
+    receiver: &Receiver<WorkingBankEntry>,
+    process_shreds_stats: Option<&mut ProcessShredsStats>,
+) -> Result<ReceiveResults> {
```

Comment:
Why not put this value in `ReceiveResults` similar to `ReceiveResults.time_elapsed`?

Reply:
done